### ISSUE TITLE
How to get sequencial result in my own dataset?

### ISSUE THREAD

**Issue by hno3sho** [fizyr/keras-retinanet#52]
Hi.
I have some trouble in prediction. Could you help me?

I have trained the RetinaNet in my own dataset and want to get the predicted result.
I made a new generator for my dataset based on PascalVocIterator, and it works well in training step.
For the prediction, I tried to get the result based  ipython example.
I would like to get the prediction result in the order of my test data path list, but the prediction is not sync to my list order.
How can I get the the result in the order of my list?

**Comment by hgaiser** [comment]
You could try passing `shuffle=False` to the generator, but in general for testing I avoid using the `next()` method. You could have a look at `evalualte_coco.py` for an example for what I mean.

**Comment by hno3sho** [comment]
I understand what you mean. I will not use the next() method.
Thanks :)
